docs/GUIDE.md

```diff
@@ -0,0 +1,137 @@
+# VisionPro Meets the World: Video Chat with Agora's Video SDK
+
+ ![](/docs/images/blog_banner.webp "banner")
+
+The future is here, and you're holding it – the Apple VisionPro device. It's a marvel of technology, but sometimes, you just need to see a friendly face across the digital divide. That's where Agora's ubiquitous Real-Time Communication (RTC) SDK comes in. With Agora, video chat on your VisionPro device transcends physical boundaries. You can connect with other VisionPro users, and Agora's expansive reach allows you to chat seamlessly with people on entirely different platforms. Imagine video calling your friends and family on their smartphones, tablets, or laptops – all through the power of Agora on your VisionPro. This quick start guide will equip you to unlock the potential of Agora's native RTC SDK and transform your VisionPro into a powerful video chat hub. 
+
+> If you are a Unity developer, you may refer to [this blog](https://www.agora.io/en/blog/vision-pro-unity-quickstart-with-agora-sdk/) for the corresponding quick start guide.
+
+
+## Prerequisites:
+
+-   Apple VisionPro    
+-   Apple computer with Silicon CPU Chip (M1 or newer generation)
+-   Xcode (15.4)  with VisionOS Support
+-   Agora Developer account
+    
+## Setups
+
+1.  Clone the GitHub project from [this location](https://github.com/AgoraIO-Community/visionOS-Quickstart).
+2. Open the project with Xcode.
+3. Import the Agora SDK plugin. The project includes the Agora SDK as a dependency package. As soon as Xcode loads the project, the package download should start automatically and finish in a few minutes.
+
+If the package did not load for some reason, e.g. networking issues, or you are setting up a brand new project of your own, see the following information to cover the package manager, CDN, and cocoapods.
+
+1. To manually add the package to your Xcode project, put the specific branch of the git location as the dependency:
+![](/docs/images/blog_avp1_branch.png "package location")
+2.  CDN download     [https://download.agora.io/null/Agora_Native_SDK_for_iOS_rel.v4.2.6.133_39484_FULL_20240126_0204_291749.zip](https://download.agora.io/null/Agora_Native_SDK_for_iOS_rel.v4.2.6.133_39484_FULL_20240126_0204_291749.zip)
+    
+3.  Cocoa Pods
+```Makefile  
+	pod 'AgoraRtcEngine_Special_iOS', '4.2.6.133.VISION'
+```
+## Running the App
+
+Fill in your test mode Agora app ID in the appID field in VisionVideoCallView.swift.
```

Comment:
To make the directions more clear, consider re-ordering the statements and separating out the "test mode" element. The "test mode" is pretty important, because if security is enabled the demo doesn't work. In the current sentence its easy for the user to miss.

To give some context, something like this:

> Open `VisionVideoCallView.swift.` and fill in your Agora app ID in the `appID` field. 

> > This guide is made for Agora projects without security enabled.

The note could use some better explanation but it at least makes it clear that they shouldn't have token security enabled.